.changeset/config.json

```diff
@@ -4,7 +4,7 @@
     "commit": false,
     "linked": [],
     "access": "public",
-    "baseBranch": "main",
+    "baseBranch": "origin/main",
```

Comment:
This fixes multiple issues. 1) Locally where there are multiple remotes referring to `main` (e.g. forks) 2) Some tools require the fully qualified branch name